src/runtime/virtcontainers/sandbox.go

```diff
@@ -37,9 +37,9 @@ import (
 	pbTypes "github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols/grpc"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/annotations"
-	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/cgroups"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/compatoci"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/cpuset"
+	resCtrl "github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/resourcecontrol"
```

Comment:
No change needed, but curious -- any reason you think this should live here at ./virtcontainers/pkg/resourcecontrol instead of at ./pkg/resourcecontrol?

Reply:
No specific reasons, besides, as you write, the fact that it was there in the first place.
I think the main reason for moving it to `runtime/pkg` is to have one common places for all runtime packages, and that's a good enough reason. I added a top commit for that purpose, if we decide we don't want to move it there, I can pop it up.